start

```diff
@@ -68,6 +68,7 @@ docker run -d --rm \
 -e MODEL_PROVIDER=http://refinery-model-provider:80 \
 -e WS_NOTIFY_ENDPOINT="http://refinery-websocket:8080" \
 -e NEURAL_SEARCH=http://refinery-neural-search:80 \
+-e INFERENCE_DIR=/inference \
```

Comment:
why is this an env var? The volume links it to static /inference so even if we change it here it will result in errors?

Reply:
also docker compose in dev-setup & release pr dont have this env var